src/os/bluestore/BlueStore.cc

```diff
@@ -1618,10 +1618,11 @@ bool BlueStore::OnodeSpace::map_any(std::function<bool(OnodeRef)> f)
   return false;
 }
 
-void BlueStore::OnodeSpace::dump(CephContext *cct, int lvl)
+template <int LogLevelV = 30>
```

Comment:
@rzarzynski could you rebase your change against master, as https://github.com/ceph/ceph/pull/19843/commits/8471cb42029339b2838f8bbe907110e9cc2253a0#diff-a9faffcf40600fd57aea5451cef5abe9R6344 in #19843 was merged recently. and your PR changes the function signature of `BlueStore::_dump_onode()`.

Reply:
Updated.